src/main/kotlin/com/dp/advancedgunnerycontrol/weaponais/tags/OpportunistTag.kt

```diff
@@ -23,8 +25,7 @@ class OpportunistTag(weapon: WeaponAPI) : WeaponAITagBase(weapon) {
     }
 
     override fun shouldFire(entity: CombatEntityAPI, predictedLocation: Vector2f): Boolean {
-
-        if(!determineIfShotWillHit(predictedLocation, entity.collisionRadius.times(0.5f), weapon)){
+        if(isAimable(weapon) && !determineIfShotWillHit(predictedLocation, effectiveCollRadius(entity), weapon)){
```

Comment:
`isAimable(weapon)` allows to fire missiles at targets that are outside weapon mount firing arc. Otherwise `Opportunist` tag prevented my Breach SRM from engaging targets at weird angles, eg backwards. Or was that the intention of `Opportunist` tag?